lib/block-directory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Block directory functions.
+ *
+ * @package gutenberg
+ */
+
+if ( ! has_action( 'admin_enqueue_scripts', 'gutenberg_aenqueue_block_editor_assets_block_directory' ) ) {
```

Comment:
I guess we have a typo here. Mayve tt should be gutenberg_enqueue_block_editor_assets_block_directory instead of  gutenberg_aenqueue_block_editor_assets_block_directory?

Reply:
Good catch, updating right away.

Reply:
It should be `enqueue_block_editor_assets_block_directory` because we won't use the `gutenberg_` prefix in core.